src/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs

```diff
@@ -106,13 +106,17 @@ public void DnsObsoleteGetHostByName_IPv6String_ReturnsOnlyGivenIP()
         public void DnsObsoleteGetHostByName_EmptyString_ReturnsHostName()
         {
             IPHostEntry entry = Dns.GetHostByName("");
+
+            // Dns ASCII comparisons are case insensitive. Refer RFC4343
```

Comment:
I would rephrase to this:

"DNS labels should be compared as case insensitive for ASCII characters.  See RFC 4343."